app/containers/Facilities/Facility.tsx

```diff
@@ -8,18 +8,17 @@ export const FacilityComponent = (props) => {
     return null;
   }
 
-  if (
-    !facility.facilityName
-      .toLowerCase()
-      .includes(props.facilityInput.toLowerCase())
-  )
+  let {facilityName} = facility;
+
+  if (!facility.facilityName) facilityName = 'UNDEFINED FACILITY NAME';
```

Comment:
maybe replace this with `'empty name (swrs facility id = ....)'` so that the user can identify the facility if needed